apps/src/applab/applab.js

```diff
@@ -782,7 +776,8 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugSlider: showDebugConsole,
-    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers')
+    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers'),
+    debuggerUi,
```

Comment:
I'm still confused about why this needs to be in Redux - so that we can hide it (switch to design mode) and re-render it with state when we switch back?  The command history definitely seems like it should be stored in redux by itself.  The JsInterpreter instance, I think, is only created when we execute a command by cloning the main JSInterpreter instance and running the one statement - I'm not sure we need to keep it around.